packages/desktop/package.json

```diff
@@ -63,7 +63,7 @@
         ],
         "appId": "org.iota.firefly",
         "afterSign": "./scripts/notarize.macos.js",
-        "asar": false,
+        "asar": true,
```

Comment:
Is this correct? Because in the commit "82777f1fae0e339260c28308faa38b1808ed5619" it was changed to false...

Reply:
afaik it was changed by @laumair just for testing purposes

Reply:
We should keep it enabled as it helps get around problems posed by the Windows path length limit